blocks/model.py

```diff
@@ -190,7 +181,18 @@ def get_params(self):
         parameters that do not belong to any brick.
 
         """
-        return self.params
+        brick_parameter_names = {
+            v: k for k, v in Selector(
+                self.top_bricks).get_parameters().items()}
+        parameter_list = []
+        for parameter in VariableFilter(roles=[PARAMETER])(
+                self.shared_variables):
```

Comment:
I know it's from old code, but I would use ```self.parameters``` directly rather than ```VariableFilter(roles=[PARAMETER])(self.shared_variables)```.